src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -95,11 +101,15 @@ static void DeleteOtherPlatformHost(Permutation permutation)
                 x86.Delete();
             }
         }
-
-        [TestFixtureSetUp]
-        public void RunBeforeAnyTests()
+    }
+    
+    [SetUpFixture]
+    public class TestSessionInitializer
+    {
+        [SetUp]
+        public void Setup()
         {
-            SessionId = Guid.NewGuid().ToString();
+            Base.SessionId = DateTime.UtcNow.Ticks.ToString();
```

Comment:
This is to generate a build ID if you will, where we run all the tests. To have a Test or Fixture ID we can have the same approach but generate them per test. What I wanted to do here was to separate latest 'batch' of tests from all the others builds/test run on the same day or previous days.